packages/editor/src/client/routes/imageList.tsx

```diff
@@ -127,10 +127,13 @@ export function ImageList() {
       {images.length > 0 ? (
         <FlexboxGrid justify="space-around" style={{marginTop: '20px'}}>
           {images.map((image, key) => (
-            <FlexboxGrid.Item colspan={7} style={{marginBottom: '20px'}} key={key}>
+            <FlexboxGrid.Item colspan={7} style={{marginBottom: '20px', maxWidth: '300'}} key={key}>
               <Link route={ImageEditRoute.create({id: image.id}, current ?? undefined)}>
-                <Panel shaded bordered bodyFill>
-                  <img src={image.thumbURL || ''} />
+                <Panel shaded bordered bodyFill style={{height: '200', width: 'calc(100% + 2px)'}}>
+                  <img
+                    src={image.mediumURL || ''}
```

Comment:
@0815pascal  I am not sure how thumbnail and medium etc. URLs are being generated for images but if there is a thumbURL? and if not I guess we may show a placeholder image :S what do you think?

Reply:
Yes it's true that there is also a `thumbURL` but I figured that `thumbURL` causes the picture to be cropped which sometimes results in weird looking images. If you go to `packages/editor/src/client/image.graphql` you can see how the different ImageURLs are generated: 

```
mediumURL: transformURL(input: {width: 300})
thumbURL: transformURL(input: {width: 280, height: 200})
```
The advantage of using `mediumURL` in my mind is that it only reduces *width* and so it's scaled down proportionally. When using `thumbURL` we're forced into using certain *width* and *height* which causes the image to be cropped. 

But then again I might have misunderstood something here...

Reply:
Thanks for the explanation, I understand now :+1: